recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "tifffile" %}
-{% set version = "2021.11.2" %}
-{% set sha256 = "153e31fa1d892f482fabb2ae9f2561fa429ee42d01a6f67e58cee13637d9285b" %}
+{% set version = "2023.4.12" %}
```

Comment:
Why not updating to 2023.8.30 at this point?
2023.8.30 is officially compatible with 3.12 too.

Reply:
If I'm not mistaken, newer versions drop support for Python 3.8.

Reply:
The latest we can go is v2023.7.10.

Reply:
That would make sense, but it doesn't prevent us from adding newer versions as well.

Reply:
We could update to 2023.7.10. Or we update to the latest and drop support for 38 for this package. But I think it's a snowflake package, so maybe we can't do that...

Reply:
Or I think it's a ❄️ package...

Reply:
It's not snowflake, just a ticket that was left in progress for many sprints.

Reply:
looking at 2023.8.30, it does not support python 3.8
https://github.com/cgohlke/tifffile/blob/103b6a337db6a84293562bda65fdc8c382281fe2/setup.py#L86C2-L86C29
also newer image codecs
imagecodecs>=2023.8.12',
https://github.com/cgohlke/tifffile/blob/103b6a337db6a84293562bda65fdc8c382281fe2/setup.py#L93C14-L93C38

Therefore I would merge as it is